services/sasquatch/charts/strimzi-kafka/templates/connect.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: {{ .Values.cluster.name }}
+  annotations:
+    # Use Connect REST API to configure connectors
+    strimzi.io/use-connector-resources: "false"
+spec:
+  version: {{ .Values.kafka.version | quote }}
+  replicas: {{ .Values.connect.replicas }}
+  bootstrapServers: {{ .Values.cluster.name }}-kafka-bootstrap:9093
+  tls:
+    trustedCertificates:
+      - secretName: {{ .Values.cluster.name }}-cluster-ca-cert
+        certificate: ca.crt
+  authentication:
+    type: tls
+    certificateAndKey:
+      secretName: {{ .Values.cluster.name }}-connect
+      certificate: user.crt
+      key: user.key
+  config:
+    group.id: {{ .Values.cluster.name }}-connect
+    offset.storage.topic: {{ .Values.cluster.name }}-connect-offsets
+    config.storage.topic: {{ .Values.cluster.name }}-connect-configs
+    status.storage.topic: {{ .Values.cluster.name }}-connect-status
+    # -1 means it will use the default replication factor configured in the broker
+    config.storage.replication.factor: -1
+    offset.storage.replication.factor: -1
+    status.storage.replication.factor: -1
```

Comment:
I think I'd find this easier to read if it were written in normal YAML syntax, e.g.:
```
offset:
  storage:
    topic: {{ .Values.cluster.name }}-connect-offets
    replication:
      factor: -1
# ...
```

Reply:
Oh, those are Kafka configuration options, it didn't occur to me I could use normal YAML syntax here, I'll try.